src/model/Note.cpp

```diff
@@ -0,0 +1,82 @@
+#include <set>
+#include <string>
+
+#include "model/Note.hpp"
+#include "model/ModelClock.hpp"
+#include "model/NoteId.hpp"
+#include "model/TagId.hpp"
+#include "model/Timestamp.hpp"
```

Comment:
If it's a source file, we sorting a bit different.
```
a header associated with this source file

(a blank line)

std:: headers alphabetically

(a blank line)

our project headers alphabetically

```

For the current case, it will be:
```c++
#include "model/Note.hpp"

#include <set>
#include <string>

#include "model/ModelClock.hpp"
#include "model/NoteId.hpp"
#include "model/TagId.hpp"
#include "model/Timestamp.hpp"
```

Reply:
done